### ISSUE TITLE
Replay Improvements

### ISSUE THREAD

**Issue by TeslaTrooper09** [Robosturm/Commander_Wars#813]
1. Turn and Action rewind buttons please.
2. please allow for access to the full options menu.
3. allow for the showing of victory conditions.
4. option for camera to follow cursor, if i have it zoomed 100%, the camera will not follow the cursor and will perform actions outside the field of view.
5. add the option to have an adjustable pause period between turn starts and unit actions, with faster animations it becomes a animation simulator.
6. hide replay bar when not hovering over it.
7. don't exit the map screen when the game ends
8. have a overworld victory screen with options to view the extended stats, or exit to menu
9. for the unit stats tab, track units produced and destroyed by each army and displayed by the associated colors.
10. add an intermediate zoom between the two current ones.

Thanks in advance!

**Comment by Robosturm** [comment]
1. Action Rewind --> won't be done. Technically this isn't possible
2. Full-Options won't be available. Only the game menu options can be made available
3. Possible and a good idea
4. This should already be possible once 2 is done. Since cursor following is a normal option
5. Low Priority compared to the rest
7. Already done in the pre-release
8. Won't be added only a path to the normal victory menu
9. Mid Priorioty
10. Not possible --> possibly creates weird graphic glitches due to sprites scaling with values not a 2^n

**Comment by TeslaTrooper09** [comment]
cool. thanks!

**Comment by Robosturm** [comment]
Turn Rewind added.
